genesis/camino_genesis.go

```diff
@@ -128,9 +129,16 @@ func validateCaminoConfig(config *Config) error {
 	}
 
 	// validate msig aliases
-	for idx, msig := range config.Camino.InitialMultisigAddresses {
+	for _, msig := range config.Camino.InitialMultisigAddresses {
+		if msig.MemoBytes == nil || len(msig.MemoBytes) == 0 {
```

Comment:
Please remove this part / check, even we write a number into it, this must not be true for others.
The only check which is important later is that generated idx don't have dups.